editoast/src/main.rs

```diff
@@ -381,6 +385,28 @@ async fn import_rolling_stock(
     Ok(())
 }
 
+async fn clone_infra(
+    infra_args: InfraCloneArgs,
+    pg_config: PostgresConfig,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let db_pool = Data::new(get_pool(pg_config.url()?, pg_config.pool_size));
+    match Infra::clone(infra_args.id, db_pool, infra_args.new_name).await {
+        Ok(cloned_infra) => println!(
+            "✅ Infra {} (ID: {}) was successfully cloned",
+            cloned_infra.name.unwrap(),
+            cloned_infra.id.unwrap()
+        ),
+        Err(e) => {
+            if e == InfraError::NotFound(infra_args.id).into() {
+                eprintln!("❌ {}", e.message);
+                return Ok(());
+            }
```

Comment:
We need to return an error otherwise code exit will be `0`.

```bash
$ editoast infra clone 42 # Infra 42 doesn't exists
$ echo $? # Expected not 0
```

Note: if you want a nice error output you can simply `exit(1)` after printing your error message.

Reply:
Done.